Storage/Storage/Model/GeneralAppSettings.swift

```diff
@@ -0,0 +1,26 @@
+
+import Foundation
+
+/// An encodable/decodable data structure that can be used to save files. This contains
+/// miscellaneous app settings.
+///
+/// Sometimes I wonder if `AppSettingsStore` should just use one plist file. Maybe things will
+/// be simpler?
```

Comment:
I'm not sure if this wanted to be a temporary comment, or if you want to remember it :D

Reply:
It's intentional. Maybe somebody will respond at some point. 🤣